### DISCUSSION TITLE
Is Libreddit cut by Reddit's New API?

### DISCUSSION THREAD

**Discussion by MarcoBrandao** [spikecodes/libreddit#825] [General]
I'm getting on my browser this message:

**`Couldn't send request to Reddit: error trying to connect: dns error: failed to lookup address information: Try again: Head back home?`**

Today we hit the June 19th date mentioned in this post: https://github.com/libreddit/libreddit/issues/785#issue-1674613885

Are we out?

**Comment by sigaloid** [comment]
Still works for me - it sounds like a DNS error, not an unauthorized response. Have you tried checking a different instance?

**Reply by MarcoBrandao** [reply]
Hum, maybe I have done something wrong. Let me reboot my server.

**Reply by MarcoBrandao** [reply]
The reboot fixed the problem.
Nothing to see here.
Sorry.

What got me was the June 19th date. I thought it was done.
